sunpy/map/tests/test_mapbase.py

```diff
@@ -1701,10 +1699,10 @@ def test_map_arithmetic_addition_subtraction(aia171_test_map, value):
 
 
 @pytest.mark.parametrize('value', [
-    10 * u.ct,
-    u.Quantity([10], u.ct),
-    u.Quantity(np.random.rand(128), u.ct),
-    u.Quantity(np.random.rand(128, 128), u.ct),
+    10 * u.s,
+    u.Quantity([10], u.s),
+    u.Quantity(np.random.rand(128), u.s),
+    u.Quantity(np.random.rand(128, 128), u.s),
     10.0,
```

Comment:
Why have `ct` changed to `s`?  Rather than `DN`?

Reply:
Fair question. Because it seemed silly to be multiplying (or dividing) `DN` by something in `ct`